lib/run-compilation.js

```diff
@@ -14,6 +14,7 @@ var errorMessage = require('./constants').errorMessage;
 module.exports = function runCompilation (options, compiler, done) {
   var errors = [];
   var warnings = [];
+  var compilerMethod = options.emitErrors ? 'after-compile' : 'after-emit';
```

Comment:
`cross-env` is a good idea. Thanks for that, I think that we could have them in a separate PR if that's not too much trouble, otherwise you can throw them in here, I really don't mind. 👍 

side note, something to be aware of: https://iamakulov.com/notes/npm-malicious-packages/